scss/_settings_colors.scss

```diff
@@ -12,21 +13,20 @@ $color-mid: #999 !default;
 $color-mid-dark: #666 !default;
 $color-dark: #111 !default;
 $color-x-dark: #000 !default;
-
-$color-negative: #c7162b !default;
-$color-caution: #f99b11 !default;
-$color-positive: #0e8420 !default;
-$color-information: #335280 !default;
+$color-input-shadow: rgba($color-x-dark, 0.12) !default;
 
 $color-label-validated: #006b75;
 
-$color-input-shadow: rgba($color-x-dark, 0.12) !default;
-
-$color-accent: $color-brand !default;
-$color-accent-background: $color-accent !default;
+$color-negative: #c7162b !default;
+$color-caution: #f99b11 !default;
+$color-positive: hsl(129, 81%, 29%) !default;
 
-// for visual outline on :active / :focus elements
-$color-focus: #19b6ee !default;
+$color-link: #06c !default;
```

Comment:
Please use fully qualified hex values please `#0066cc`

Reply:
sure, changed. Can't help but wonder though, vscode underlined it in red, and suggested the shorthand. We're also using shorthand for fff, 111, etc - so why should this be the full version?

Reply:
I believe our linter enforces short hex values if they are possible. This was the case both with previous sass-lint and current stylelint.
As Lyubo pointed out, we have shorthand for some colors (`#111`, `#666`, `#fff`, `#999`).